components/resources/permissions_strings.grdp

```diff
@@ -21,4 +21,10 @@
   <message name="IDS_PERMISSIONS_BUBBLE_SITE_PERMISSION_LINK" desc="Site permission link in the footnote description.">
     site permission
   </message>
+  <message name="IDS_OPEN_AI_CHAT_PERMISSION_FRAGMENT" desc="Label for Leo AI chat permission prompt.">
+    Allow to open Leo in Brave?
```

Comment:
This string reads slightly weird. Additionally, other permissions normally read

"{site} wants to {do thing}"

![image](https://github.com/user-attachments/assets/4b285252-e4b4-435f-8014-cceaa8dfb144)

I think this string should be "Open Leo in Brave"

Reply:
This is another permission prompt:
![image](https://github.com/user-attachments/assets/26a8d9b9-548a-4e80-b68c-d60659615e3c)

Reply:
@ShivanKaul should review

Reply:
The grammar seems a bit strange here, I suggest "Allow opening Leo in Brave" since that's what happens after the user clicks Allow.

Reply:
Also I think we want these strings in a new permission strings file similar to https://github.com/brave/brave-core/blob/master/components/resources/localhost_access_permission_strings.grdp and others included in https://github.com/brave/brave-core/blob/master/components/resources/brave_components_strings.grd#L779-L805.

Reply:
`components/resources/permission_strings.grdp` seems to make sense to me, seems matching what's in chromium's `components/permissions_strings.grdp`. It's a bit weird imho to have one file per new permission with very little strings inside.
As for the text, was following what's written in the spec, will update to "Allow opening Leo in Brave" on desktop.

Reply:
i unresolved this thread because it seems the text still needs updating

Reply:
Oops, missed this, https://github.com/brave/brave-core/pull/26082/commits/0a9c5a2703edf5b28264cff4a2e09eeda821c16d